docs/backend/previewing-emails.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Previewing Emails
+---
+
+# Previewing emails in development
+
+You can modify the test in `/test/mailers/previews`
+You can view the previews at (for example) `http://localhost:3000/rails/mailers/notify_mailer/new_reply_email`
```

Comment:
I think you can just say something like: "You can find all the registered previews at `http://localhost:3000/rails/mailers`"